programs/mpl-core/Cargo.toml

```diff
@@ -13,13 +13,13 @@ crate-type = ["cdylib", "lib"]
 borsh = "^0.10"
 shank = "0.4.2"
 modular-bitfield = "0.11.2"
-mpl-bubblegum = "1.4.0"
+mpl-bubblegum = "2.1.0"
 num-derive = "^0.3"
 num-traits = "^0.2"
-solana-program = "^1.17"
+solana-program = "2.0"
```

Comment:
Would setting this to latest (just the program, not the SDK) work? If it's going to take more than an hour to fix dependencies it's not worth it, but just curious if doing 2.2.18 works off the bat it's probably worth doing.